.github/workflows/test.yml

```diff
@@ -150,14 +155,16 @@ jobs:
           TEST: ${{ matrix.test }}
           VERSION: ${{ github.head_ref || 'master' }}
           CODALAB_LINK_MOUNTS: /tmp
-      - uses: medyagh/setup-minikube@latest
+      - uses: actions/setup-go@v3
         if: matrix.runtime == 'kubernetes'
+        with:
+          go-version: '1.18.1'
       - name: Run tests using Kubernetes runtime
         if: matrix.runtime == 'kubernetes'
         run: |
           sh ./tests/test-setup.sh
           sh ./scripts/local-k8s/setup-ci.sh
-          #python3 test_runner.py --version ${VERSION} ${TEST}
+          # python3 test_runner.py --version ${VERSION} ${TEST}
```

Comment:
Shouldn't this be uncommented so that tests will use kubernetes runtime?

Reply:
When I look at the logs, it looks like the kubernetes runtime tests might only have executed the test setup (and not the actual tests) because this line is commented. But I could be wrong?

Reply:
Correct -- I commented this out to debug a CI issue -- and good catch! Re-added it now.